grails-app/taglib/org/pih/warehouse/GoogleAnalyticsTagLib.groovy

```diff
@@ -0,0 +1,77 @@
+package org.pih.warehouse
+
+import org.codehaus.groovy.grails.commons.ConfigurationHolder
+
+class GoogleAnalyticsTagLib {
+
+    static namespace = "g"
+
+    /**
+    * Injecting gtag:
+        <!-- Google tag (gtag.js) -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id=TRACKING_ID"></script>
+        <script>
+            window.dataLayer = window.dataLayer || [];
+            function gtag(){dataLayer.push(arguments);}
+            gtag('js', new Date());
+
+            gtag('config', TRACKING_ID);
+        </script>
+    * */
+    def gtag = {attrs ->
+        if (!getTrackingIds()) {
+            out << ""
```

Comment:
is there a reason we're pushing an empty string here?

Reply:
No reason, we can push nothing.